### PR TITLE
Create Project 3 area and migrate Confluence docs

### PR DESCRIPTION
**Pull request by brendankntb**
This PR starts to populate the Project 3 Athlete Wearable Tech section of the documentation site.

### CODE REVIEW

**Review by SassafrasAU — APPROVED** [review]

File: docs/project-3/index.md
Comment (MorgaineBarter) [inline docs/project-3/index.md:1]: as*

**Review by MorgaineBarter — COMMENTED** [review]
there's a typo in the index.md
the rest look good.

**Comment by SassafrasAU** [comment]
Quickly fixed the typo with the changes I pushed

File: docs/project-3/index.md
Comment (MorgaineBarter) [inline docs/project-3/index.md:1]: still shows as ad not as

**Review by MorgaineBarter — COMMENTED** [review]
index.md file typo still not fixed

File: docs/project-3/index.md
Comment (SassafrasAU) [reply docs/project-3/index.md:1]: Sorry I should have clarified, the typo fix was changed on the main repo, not this fork (https://github.com/Redback-Operations/redback-documentation/blob/main/docs/project-3/index.md)

**Pull request merged**